README.md

```diff
@@ -119,6 +123,22 @@ For details, have a look at the Makefile.
 If you want to disable the SSO for this session, click on the tray icon and select the guest account.
 In case you are already logged in, you might need to clear all cookies on `login.microsoftonline.com`.
 
+### Single Page Applications
```

Comment:
@jan-kiszka As discussed offline, I added a note regarding how to enable the Background SSO. I hope this helps.